apps/api-documenter/src/documenters/MarkdownDocumenter.ts

```diff
@@ -1152,10 +1157,22 @@ export class MarkdownDocumenter {
     }
   }
 
+  private _writeAlphaWarning(output: DocSection): void {
+    const configuration: TSDocConfiguration = this._tsdocConfiguration;
+    const betaWarning: string =
+      'This API is provided as an alpha preview for developers and may change' +
```

Comment:
I updated the phrasing for both `alpha` and `beta` alerts to help differentiate them a bit. This may not be necessary since we also annotate the headings with `(ALPHA)` and `(BETA)`. Happy to make whatever changes the team recommends here 🙂